lib/actions/deploy.coffee

```diff
@@ -112,6 +126,11 @@ module.exports =
 			parameter: 'source'
 			description: 'The source directory to use when building the image'
 			alias: 's'
+		},
+		{
+			signature: 'nologs'
```

Comment:
I'm not convinced by the name - this one makes me think that it would hide local logs - maybe `no-logs-upload`?